iModelCore/GeomLibs/geom/test/CurvePrimitiveTest/t_IGeometry.cpp

```diff
@@ -113,13 +113,13 @@ void DoRoundTrip (IGeometryPtr g0, bool emitGeometry, int serializerSelect)
     if (checkPolyfaceQueryCarrier && polyface.IsValid () && buffer0.size () > 8)
         {
         PolyfaceQueryCarrier carrier0 (0, false, 0, 0, nullptr, nullptr);
-        Check::True(BentleyGeometryFlatBuffer::BytesToPolyfaceQueryCarrier(&buffer0[0], buffer0.size(), carrier0));
+        Check::True(BentleyGeometryFlatBuffer::BytesToPolyfaceQueryCarrierSafe(&buffer0[0], buffer0.size(), carrier0));
 
         bvector<Byte> buffer1;
         BentleyGeometryFlatBuffer::GeometryToBytes (*polyface, buffer1);
 
         PolyfaceQueryCarrier carrier1 (0, false, 0, 0, nullptr, nullptr);
-        Check::True(BentleyGeometryFlatBuffer::BytesToPolyfaceQueryCarrier(&buffer1[0], buffer0.size(), carrier1));
+        Check::True(BentleyGeometryFlatBuffer::BytesToPolyfaceQueryCarrierSafe(&buffer1[0], buffer0.size(), carrier1));
```

Comment:
Make sure you apply all the _later_ commits from imodel02.

```suggestion
        Check::True(BentleyGeometryFlatBuffer::BytesToPolyfaceQueryCarrierSafe(&buffer1[0], buffer1.size(), carrier1));
```